components/skeleton/skeleton-mixin.js

```diff
@@ -155,7 +156,11 @@ export const SkeletonMixin = dedupeMixin(superclass => class extends RtlMixin(su
 			 * Renders the input as a [skeleton loader](https://github.com/BrightspaceUI/core/tree/main/components/skeleton)
 			 * @type {boolean}
 			 */
-			skeleton: { reflect: true, type: Boolean  }
+			skeleton: { reflect: true, type: Boolean  },
+			skeletonDisplay: { reflect: true, type: Boolean, attribute: 'skeleton-display' },
+			_parentSkeleton: { state: true },
```

Comment:
Does `_parentSkeleton` need to be a state property, or could it just be a regular private variable? The only advantage to it being state would be that changing it will trigger a re-render, but it seems like we only care about re-rendering when `skeleton` or `skeletonDisplay` change.

Similarly for `_readyToDisplay` and `_waitToDisplay` -- I don't think they need to reflect to attributes and can also just be private variables.

Reply:
Ah yeah, I forgot to take out `reflect` stuff...I originally was playing around directly with these variables.

When you say private variable, do you mean setting them as `this._x`, but not declaring in properties?

Reply:
Exactly yeah -- just set it in the constructor.